framework/messages/el/yii.php

```diff
@@ -72,6 +71,7 @@
     'Yii Framework' => 'Yii Framework',
     'You are not allowed to perform this action.' => 'Δεν επιτρέπεται να εκτελέσετε αυτή την ενέργεια.',
     'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'Μπορείτε να ανεβάσετε το πολύ {limit, number} {limit, plural, one{αρχείο} few{αρχεία} many{αρχεία} other{αρχεία}}.',
+    'You should upload at least {limit, number} {limit, plural, one{file} other{files}}.' => 'Θα πρέπει να ανεβάσετε τουλάχιστον {limit, number} {limit, plural, one{file} other{files}}.',
```

Comment:
Seems not to be fully translated.

Reply:
Need some help with that one. Do I translate the text inside {}? Eg. in the phrase `{limit, plural, one{file} other{files}}` which words do I translate?

Reply:
I think it should be `Θα πρέπει να ανεβάσετε τουλάχιστον {limit, number} {limit, plural, one{αρχείο} few{αρχεία} many{αρχεία} other{αρχεία}}`